test/src/test/java/hudson/tasks/EnvVarsInConfigTasksTest.java

```diff
@@ -92,10 +93,9 @@ public void testFreeStyleShellOnSlave() throws Exception {
 	}
 
 	public void testFreeStyleAntOnSlave() throws Exception {
-        if (jenkins.getDescriptorByType(Ant.DescriptorImpl.class).getInstallations().length == 0) {
-            System.out.println("Cannot do testFreeStyleAntOnSlave without ANT_HOME");
-            return;
-        }
+		// Cannot do testFreeStyleAntOnSlave without ANT_HOME
+	    Assume.assumeFalse(jenkins.getDescriptorByType(Ant.DescriptorImpl.class).getInstallations().length == 0);
```

Comment:
and migrate `Cannot do testFreeStyleAntOnSlave without ANT_HOME` into assume argument